camel/agents/tool_agents/hugging_face_tool_agent.py

```diff
@@ -164,7 +164,11 @@ def step(
         """
         if remote is None:
             remote = self.remote
-        return self.agent.run(*args, remote=remote, **kwargs)
+        agent_output = self.agent.run(*args, remote=remote, **kwargs)
+        from transformers.tools.agent_types import AgentImage  # type: ignore
+        if isinstance(agent_output, AgentImage):
```

Comment:
isinstance should not be there in the core code. What are the possible return types? If only AgentImage, remove this check.

Reply:
The return value can be `AgentImage`, `AgentAudio`, etc. `isinstance` is the only way we can identify them. We must identify it for further processing.